HPCC-14752 DOCS:Update Platform installation references

g-pan opened the pull request:
FIX HPCC-14752 DOCS:Update Platform installation references
Update the Platform installation references in the docs.

Signed-off-by: G Panagiotatos greg.panagiotatos@lexisnexis.com

@JamesDeFabia please review

g-pan committed:
HPCC-14752 DOCS:Update Platform installation references

FIX HPCC-14752 DOCS:Update Platform installation references
Update the Platform installation references in the docs.

Signed-off-by: G Panagiotatos <greg.panagiotatos@lexisnexis.com>

g-pan committed:
HPCC-14752 DOCS:Update Platform installation references

FIX HPCC-14752 DOCS:Update Platform installation references
Update the Platform installation references in the docs.

Signed-off-by: G Panagiotatos <greg.panagiotatos@lexisnexis.com>

HPCCSmoketest commented:
Automated Smoketest Build: success
Install hpccsystems-platform-community_6.0.0-beta1.el6.x86_64.rpm
HPCC Start: OK
HPCC Stop: OK
HPCC Uninstall: OK

g-pan committed:
HPCC-14752 DOCS:Update Platform installation references

FIX HPCC-14752 DOCS:Update Platform installation references
Update the Platform installation references in the docs.

Signed-off-by: G Panagiotatos <greg.panagiotatos@lexisnexis.com>

HPCCSmoketest commented:
Automated Smoketest Build: success
Install hpccsystems-platform-community_6.0.0-beta1.el6.x86_64.rpm
HPCC Start: OK
HPCC Stop: OK
HPCC Uninstall: OK

HPCCSmoketest commented:
Automated Smoketest Build: success
Install hpccsystems-platform-community_6.0.0-beta1.el6.x86_64.rpm
HPCC Start: OK
HPCC Stop: OK
HPCC Uninstall: OK

JamesDeFabia commented:
Looks good to me
